scripts/configs/drone/main.jsonnet

```diff
@@ -174,6 +188,24 @@ local docker_publish(repo, auth, tag, os, arch, version='') =
     ])
     + releaseOnly,
 
+    step(
+      'docker publish (with browser) tags',
+      [
+        '{ echo latest-browser,$(eval ./scripts/version)-browser ; } > .tags',  // use with-browser tags for docker plugin
```

Comment:
As a small nit, is there a reason for this command using eval and curly braces?
On my PC, this seems to work:
```console
echo "latest-browser,$(./scripts/version)-browser"
```

Reply:
That should work - updating!